src/components/ebay-breadcrumb/template.marko

```diff
@@ -0,0 +1,12 @@
+<if(data.items)>
+    <nav aria-labelledby="breadcrumbs_heading" class="breadcrumb" role="navigation" ${data.htmlAttributes} w-bind w-onclick=(data.preventDefault && 'handleClick')>
+        <h2 id="breadcrumbs_heading" class="clipped">${data.heading}</h2>
+        <ul>
+            <for(item in data.items)>
+                <li if(item)>
```

Comment:
Why the extra check? We're in a `<for>` loop so I would assume `item` is implicitly present.